opentelemetry/chart/templates/logs-collector.yaml

```diff
@@ -39,6 +39,7 @@ spec:
       port: 9999
 {{- end }}
   image: {{ index .Values "opentelemetry-operator" "manager" "collectorImage" "repository" }}:{{ index .Values "opentelemetry-operator" "manager" "collectorImage" "tag" }}
+  upgradeStrategy: automatic
```

Comment:
is this really needed? it reads as if this is the default

https://pkg.go.dev/github.com/open-telemetry/opentelemetry-operator#section-readme

> The default and only other acceptable value for .Spec.UpgradeStrategy is automatic.